Comments/components/commentIndicator.js

```diff
@@ -62,6 +63,11 @@ export class CommentIndicator extends Label {
     return pt(xRelative, yRelative);
   }
 
+  delete () {
+    disconnectAll(this);
```

Comment:
Learning-question: What does this do?

Reply:
Removes all connections established with connect(,,,) statements.